pkg/collector/corechecks/snmp/snmp_test.go

```diff
@@ -56,6 +56,8 @@ func createDeps(t *testing.T) deps {
 }
 
 func Test_Run_simpleCase(t *testing.T) {
+	testDir := t.TempDir()
+	coreconfig.Datadog.SetWithoutSource("run_path", testDir)
```

Comment:
Can we add a comment to explain why this is needed? just to be sure it's clear for future readers

Reply:
done https://github.com/DataDog/datadog-agent/pull/25887/commits/8646d253270d88608e89a5918fbd552777a32a16